app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -28,20 +25,32 @@ import org.mozilla.focus.tiles.CustomTilesManager
 import org.mozilla.focus.tiles.BundledHomeTile
 import org.mozilla.focus.tiles.HomeTile
 import org.mozilla.focus.utils.OnUrlEnteredListener
+import android.content.Context.MODE_PRIVATE
```

Comment:
I'm not sure why these imports moved from above to here, please fix your Android Studio to not autoformat these imports for no reason.